test/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "rbac-test",
+  "version": "1.0.0",
+  "description": "RBAC behavior test",
+  "author": "[Jay] <[pan@peaq.io]>",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
It's better to add the exec command on the script tag 
```
"scripts": {
    "start": "node rbac_deploy.js && node rbac_test.js",
    "deploy": "node rbac_deploy.js",
    "test": "node rbac_test.js"
 }
```

then update the README to `npm run start`